xml/slurm.xml

```diff
@@ -925,17 +1089,23 @@ Description="subgroup" Organization=bavaria</screen>
     <literal>-R ssh</literal>in the <literal>pdsh</literal>commands below. This
     is less scalable and you may run out of usable ports.
    </para>
+   <warning>
+    <title>Upgrade <literal>slurmdbd</literal> databases before other &slurm; components</title>
+    <para>
+     If <literal>slurmdbd</literal> is used, always upgrade the
+     <literal>slurmdbd</literal> database <emphasis>before</emphasis> starting
+     the upgrade of any other &slurm; component. The same database can be connected
+     to multiple clusters and must be upgraded before all of them.
```

Comment:
IIRC, @sknorr wanted this paragraph to be re-worded to make it a little less ambiguous, and we worked out something he liked better. (I am crediting him, not in any way blaming him; I did not spot the ambiguity, he did.) If you have not already, I suggest comparing with the same paragraph before the new Slurm chapter was incorporated.

Reply:
This is that very rewording, I think? It's from a commit Stefan made in the RNs, at any rate.

There's also a different, but similar, note that was updated in an earlier commit that I can confirm is still intact.

Reply:
Oh well. I have definitely seen this one before, I can say this much. *If* I remember correctly, the version in the release notes is what I came up with after discussion with Egbert wanted told me he wanted to make sure option of multi-database connections is emphasized, because users may forget that issue.

I sadly have no remembrance of telling Liam about this sentence specifically, so I have no clue what I said exactly or why (and don't want to look through blames/PRs now).  :salt: advised.